testing/jormungandr-scenario-tests/src/node.rs

```diff
@@ -7,7 +7,7 @@ use chain_impl_mockchain::{
 };
 use indicatif::ProgressBar;
 use jormungandr_integration_tests::{
-    common::jormungandr::logger::JormungandrLogger,
+    common::jormungandr::{logger::JormungandrLogger, JormungandrRest, RestError},
```

Comment:
Shouldn't we move all reusable stuff to `jormungandr-testing-utils` to reduce the cross-crate dependency spaghetti?

Reply:
agreed

Reply:
also agreed, but test project refactoring is not a highest priority right now. I want to do that in many small steps. this is intermediate step which moves some of rest methods to integration test and then move it to testing-utils.